src/main/java/me/karakelley/http/RequestHeaders.java

```diff
@@ -0,0 +1,19 @@
+package me.karakelley.http;
+
+import java.util.HashMap;
+
+public class RequestHeaders {
+  HashMap headers;
+
+  public RequestHeaders(HashMap headers) {
+    this.headers = headers;
+  }
+
+  public HashMap getHeaders() {
```

Comment:
1. `requestHeaders.getHeaders()`? Seems strange.
2. What does this abstraction bring to the table? What's its reason for existing?

Reply:
While it may be an early abstraction it also seems like it can easily be it's own entity